addons/website_forum/views/website_forum.xml

```diff
@@ -953,7 +967,7 @@
             users having a high karma can see offensive posts to moderate
             them.
         </p>
-        <form t-attf-action="/forum/#{ slug(forum) }/#{offensive and 'post' or 'question'}/#{slug(question)}/#{offensive and 'mark_as_offensive' or 'close'}" method="post" role="form" class="mt32 mb64 js_website_submit_form">
+        <form t-attf-action="/forum/#{ slug(forum) }/#{offensive and 'post' or 'question'}/#{slug(question)}/#{offensive and 'mark_as_offensive' or 'close'}" method="post" role="form" t-attf-class="mt32 js_website_submit_form #{'mb64 ' if not dialog else ''}">
```

Comment:
Let's keep it simple, the less code the better.
You can just remove the `mb64` from the code, and so you don't need a `dialog` variable :)
It is not especially ugly without the `mb64` anyway..
![image](https://user-images.githubusercontent.com/30048408/121882466-dc0de100-cd10-11eb-958d-f0ef0a6db974.png)